nuxeo-data-table/iron-data-table.html

```diff
@@ -601,7 +601,46 @@
           target.fire('selected');
           this._updateFlags();
         }
-      }
+      },
+        
+    _sortDirectionChanged: function(e) {
```

Comment:
Indeed, i just update the nuxeo-data-table element (because i want the "on-row-click-event") !
Thank you for your quick response.